Jenkinsfile

```diff
@@ -26,274 +26,280 @@
 //in experiments carried out, the security priviledges seemed to prevent
 //connection to the docker sock for much later versions
 
+timestamps {
 
-podTemplate(yaml: '''
-apiVersion: v1
-kind: Pod
-metadata: 
-    name: dind 
-spec:
-  affinity:
-    nodeAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-      - weight: 1
-        preference:
-          matchExpressions:
-          - key: palisade-node-name
-            operator: In
-            values: 
-            - node1
-            - node2
-            - node3
-  containers:
-  - name: jnlp
-    image: jenkins/jnlp-slave
-    imagePullPolicy: Always
-    args: 
-    - $(JENKINS_SECRET)
-    - $(JENKINS_NAME)
-    resources:
-      requests:
-        ephemeral-storage: "4Gi"
-      limits:
-        ephemeral-storage: "8Gi"
+    podTemplate(yaml: '''
+    apiVersion: v1
+    kind: Pod
+    metadata:
+        name: dind
+    spec:
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 1
+            preference:
+              matchExpressions:
+              - key: palisade-node-name
+                operator: In
+                values:
+                - node1
+                - node2
+                - node3
+      containers:
+      - name: jnlp
+        image: jenkins/jnlp-slave
+        imagePullPolicy: Always
+        args:
+        - $(JENKINS_SECRET)
+        - $(JENKINS_NAME)
+        resources:
+          requests:
+            ephemeral-storage: "4Gi"
+          limits:
+            ephemeral-storage: "8Gi"
 
-  - name: docker-cmds
-    image: 779921734503.dkr.ecr.eu-west-1.amazonaws.com/jnlp-did:200608
-    imagePullPolicy: IfNotPresent
-    command:
-    - sleep
-    args:
-    - 99d
-    env:
-      - name: DOCKER_HOST
-        value: tcp://localhost:2375
-    resources:
-      requests:
-        ephemeral-storage: "4Gi"
-      limits:
-        ephemeral-storage: "8Gi"
-  
-  - name: hadolint
-    image: hadolint/hadolint:latest-debian@sha256:15016b18964c5e623bd2677661a0be3c00ffa85ef3129b11acf814000872861e
-    imagePullPolicy: IfNotPresent
-    command:
-        - cat
-    tty: true  
-    resources:
-      requests:
-        ephemeral-storage: "1Gi"
-      limits:
-        ephemeral-storage: "2Gi"
+      - name: docker-cmds
+        image: 779921734503.dkr.ecr.eu-west-1.amazonaws.com/jnlp-did:200608
+        imagePullPolicy: IfNotPresent
+        command:
+        - sleep
+        args:
+        - 99d
+        env:
+          - name: DOCKER_HOST
+            value: tcp://localhost:2375
+        resources:
+          requests:
```

Comment:
```suggestion
          requests:
            resourceRequestCpu:4
```

Reply:
Aha! That's where :)

Reply:
I take it this would be in all the other repositories as well?